src/stan/variational/advi.hpp

```diff
@@ -398,6 +562,7 @@ namespace stan {
       }
 
       // Helper function: compute relative decrease between two doubles
+      // TODO(akucukelbir): move to stan math and test there
       double rel_decrease(double prev, double curr) const {
         return std::abs(curr - prev) / std::abs(prev);
```

Comment:
Two problems.

1.  This should only involve a single abs, and it should be fabs, as in `std::fabs((curr - prev) / prev)`.

2.  Because of the absolute value, it's not computing a relative decrease, so it should be renamed. Maybe relative diff?  

And a question.  This is only relative to `prev` rather than to both scales. 

And a bonus:

3.  Use doxygen doc, not line doc.